docker/build_scripts/install-libxcrypt.sh

```diff
@@ -19,21 +19,24 @@ check_sha256sum "v${LIBXCRYPT_VERSION}.tar.gz" "${LIBXCRYPT_HASH}"
 tar xfz "v${LIBXCRYPT_VERSION}.tar.gz"
 pushd "libxcrypt-${LIBXCRYPT_VERSION}"
 ./autogen.sh > /dev/null
+
 DESTDIR=/manylinux-rootfs do_standard_install \
-	--disable-obsolete-api \
-	--enable-hashes=all \
-	--disable-werror
-# we also need libcrypt.so.1 with glibc compatibility for system libraries
-# c.f https://github.com/pypa/manylinux/issues/305#issuecomment-625902928
-make clean > /dev/null
-sed -r -i 's/XCRYPT_([0-9.])+/-/g;s/(%chain OW_CRYPT_1.0).*/\1/g' lib/libcrypt.map.in
-DESTDIR=/manylinux-rootfs/so.1 do_standard_install \
-	--disable-xcrypt-compat-files \
-	--enable-obsolete-api=glibc \
-	--enable-hashes=all \
-	--disable-werror
-cp -P /manylinux-rootfs/so.1/usr/local/lib/libcrypt.so.1* /manylinux-rootfs/usr/local/lib/
-rm -rf /manylinux-rootfs/so.1
+    --disable-obsolete-api \
+    --enable-hashes=all \
+    --disable-werror
+if [ "${POLICY}" != "musllinux_1_1" ]; then 
```

Comment:
```suggestion
if [ "${POLICY:0:9}" == "manylinux" ]; then 
```

Reply:
Instead of doing this in every conditional, why don't we export a new variable somewhere which contains either `manylinux` or `musllinux`? Something like `POLICY_NAME` or `POLICY_PLATFORM`.

Reply:
> Instead of doing this in every conditional, why don't we export a new variable somewhere which contains either manylinux or musllinux? Something like POLICY_NAME or POLICY_PLATFORM.

Why not.
Just not at the dockerfile level as I'd rather not expose too many environment variables so `build_utils.sh` which is sourced in every script might be the good place to compute this (maybe `BASE_POLICY` ?)